README.md

```diff
@@ -25,7 +25,7 @@ These are the following Storybook features we attempt to uncover in this project
 | ⏳     | [`render`](https://storybook.js.org/docs/api/csf#custom-render-functions)                                                                                   |         |       |
 | ⏳     | [`play`](https://storybook.js.org/docs/writing-stories/play-function)                                                                                       |         |       |
 | ⏳     | [How to type the meta object and stories](https://storybook.js.org/docs/writing-stories/typescript)                                                         |         |       |
-| ⏳     | [JSDoc comments as descriptions](https://storybook.js.org/docs/api/doc-block-description#writing-descriptions) (from component, meta and stories)           |         |       |
+| 🟡     | [JSDoc comments as descriptions](https://storybook.js.org/docs/api/doc-block-description#writing-descriptions) (from component, meta and stories)           |         |       |
```

Comment:
I changed the structure of this table slightly, to have a "Links" column.

Could you add links to the stories (using https://main--663faba8e103e55dccd640dc.chromatic.com as a base URL) and to this PR?